public/locales/en/common.json

```diff
@@ -61,8 +61,10 @@
     "cancel-button": "Cancel",
```

Comment:
These keys should be grouped by their modal concerns. Like "modal-go-back" and "modal-idle-timeout".